lang/lang_en_ro.txt

```diff
@@ -340,7 +340,7 @@
 
 #MSG_FILAMENT_SENSOR c=20
 "Filament sensor"
-"Senzorul de filament"
+"Senz. de filament"
```

Comment:
Asta incape asa cum e acum. Nu prea are sens sa fie facut mai scurt

Reply:
Incape dar cand face testarea si rotita ce se invarte da peste "nt" din capat

Reply:
![20220115_123802.jpg](https://user-images.githubusercontent.com/56076331/149618748-c29cba3e-3b21-4836-a7f0-e23d83a89ec2.jpg)![20220115_123807.jpg](https://user-images.githubusercontent.com/56076331/149618749-d7990ba1-fc5c-4eab-9a4d-ba45f972cde1.jpg)

Reply:
Ouf. Vad acum. E o greseala la limita de caractere.

@3d-gussner Please fix the `#MSG_FILAMENT_SENSOR c=20`. It does not exist in the code. It should be `MSG_SELFTEST_FILAMENT_SENSOR` with c=17

Reply:
I will mark this as resolved as soon as @3d-gussner sees the message